### DISCUSSION TITLE
Bulk preparation of the Wave form

### DISCUSSION THREAD

**Discussion by OtaStrom** [SubtitleEdit/subtitleedit#6926] [Ideas]
When timing subtitles for a larger number of files (e.g. TV series), it would be useful to have the option to prepare audio curves for all files with just a few clicks (in a bulk). Currently, it is necessary to open each file and create the curve one by one.

**Comment by niksedk** [comment]
Check Video - Batch generate wave forms:
![image](https://github.com/SubtitleEdit/subtitleedit/assets/1261331/6dd0f896-4060-4469-a4e8-714a3cbc12c4)

Alternatively (if you have a fast Solid State Disc and a fast CPU), go to Options - Settings - Waveform:
![image](https://github.com/SubtitleEdit/subtitleedit/assets/1261331/a1a7e859-bdb8-4f71-8e1f-40d556f4513b)

**Comment by OtaStrom** [comment]
Perfect, thanks. I completely missed this feature.
